.github/workflows/docs.yml

```
@@ -26,91 +27,37 @@ on:
       - ".github/workflows/docs.yml"
 
 permissions:
-  contents: write
+  contents: read
 
 concurrency:
   group: ${{ github.workflow }}-${{ github.repository }}-${{ github.head_ref || github.sha }}
-  cancel-in-progress: true
+  cancel-in-progress: ${{github.ref != 'refs/heads/presto-0.293-clp-connector'}}
 
 jobs:
   build_docs:
-    name: Build and Push
-    runs-on: 8-core-ubuntu
-    container: ghcr.io/facebookincubator/velox-dev:centos9
+    name: Build
+    runs-on: yscope-gh-runner
```

```suggestion
    runs-on: ubuntu-latest
```

This workflow should be able to run on a GH runner.